cmd/bucket-handlers.go

```diff
@@ -380,7 +380,7 @@ func (api objectAPIHandlers) PutBucketHandler(w http.ResponseWriter, r *http.Req
 	}
 
 	// PutBucket does not have any bucket action.
-	s3Error := checkRequestAuthType(r, "", "", globalMinioDefaultRegion)
```

Comment:
As per my understanding, if we do this, it means the following `if` will not be needed